commands/output/output.go

```diff
@@ -48,6 +49,9 @@ status: %s
 %s`
 	// Template for displaying the summary of the comment threads for a review
 	commentSummaryTemplate = `  comments (%d threads):
+`
+	// Template for displaying the summary of the detached comment threads for a path
+	detachedCommentSummaryTemplate = `Comments (%d threads):
```

Comment:
I was trying to make the output for a list of comments consistent with the output for a list of reviews.

However, looking at this, it seems that the logic for the output of a list of reviews was not in the `output` package.

I've now fixed that and also changed the wording for the two different list outputs so that they are more consistent.